plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveMetadata.java

```diff
@@ -2316,14 +2324,14 @@ else if (e.getErrorCode().equals(TABLE_NOT_FOUND.toErrorCode())) {
                     ConnectorViewDefinition definition = createViewReader(metastore, session, view, typeManager)
                             .decodeViewData(view.getViewOriginalText().get(), view, catalogName);
                     // use owner from table metadata if it exists
-                    if (view.getOwner() != null && !definition.isRunAsInvoker()) {
```

Comment:
Interesting, this null check was always redundant, since the field can't be null